app/models/college.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+#
+# Model for site configuration / individual colleges. Will be used for
+# multi-tenancy in the future.
+#
+# @attr name [String] the college name
+# @attr admin_email [String] the admin e-mail for the college
+# @attr dean [String] the name of the college dean / head
+# @attr site_url [String] the full url (including http/https) for the Vesta site
+#   for the college
```

Comment:
also need housing rep emails and link to housing rules

we should also note in the "student experience" issue that we need to add these things to this model:
* box link to floorplans
* some sort of static text that is displayed to students

(noting this here & will comment on the issue when I'm done)